cli/skills/rules/troubleshooting.md

```
@@ -0,0 +1,63 @@
+
+### "Video file not found"
+Video paths in projects are absolute. If you moved the video:
+```bash
+# Re-create the project with the new path
+openscreen --json project create --video /new/path/recording.webm --output demo.openscreen
+```
+
+### "Invalid project data"
+The project file is missing required fields. It needs: `version`, `media` (or `videoPath`), and `editor`.
+
+### Render commands fail with "requires Electron headless bridge"
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)